mehari/router_impl.ml

```diff
@@ -19,6 +19,8 @@ module type S = sig
 
   val scope :
     ?rate_limit:rate_limiter -> ?mw:middleware -> string -> route list -> route
+
+  val virtual_hosts : ?default:handler -> (string * handler) list -> handler
```

Comment:
Maybe remove `default` paramater, does not make sense with virtual hosting at the TLS-layer using SNI